html/handykosten/gene_func.js

```diff
@@ -6,12 +6,18 @@ var tarifDatBas = [{
         mon_Kost: 7.99,
         zahlbeginn: 1,
         dat_vol: 2000,
-        sms: 'Ja',
+        sms: 'Ja', // durch boolean ersetzen.
```

Comment:
Kann dann einfacher ausgewertet werden.
War sowieso ultra dämlich, dass mit 'Ja'/'Nein'-String zu machen.
```suggestion
        sms: 'Ja', // durch boolean ersetzen.
```